SPECS/dracut/20overlayfs/overlayfs-mount.sh

```diff
@@ -53,6 +52,60 @@ parse_kernel_cmdline_args() {
     [ -z "${overlayfs}" ] && overlayfs=$(getarg rd.overlayfs=)
 }
 
+mount_overlayfs() {
```

Comment:
Please move `mount_overlayfs()` to be below `create_overlayfs()`. Moving code makes it harder to review.